Infrastructure/deps/GLFW/include/glfw3native.h

```diff
@@ -0,0 +1,356 @@
+/*************************************************************************
```

Comment:
oh wait. i guess I pasted my own copy trying to make it work... no problem I'll have it removed.